packages/react-server/core/TritonAgent/Cache.js

```diff
@@ -1,320 +1,10 @@
 
 var logger = require('../logging').getLogger(__LOGGER__)
 ,	Q = require('q')
-,	{ mixin } = require("./util")
-,	isEqual = require("lodash/lang/isEqual")
 ,	isArray = require("lodash/lang/isArray")
+,	CacheEntry = require("./CacheEntry")
 ;
 
```

Comment:
Below here is all just moved to a dedicated module so `CacheEntry` can be imported into `Request`.